HW6/IMDB.py

```diff
@@ -0,0 +1,50 @@
+imdb = open('./ratings.list', 'r')
+file_str = imdb.read()
+start_point = file_str.find('      00000')
+end_point = file_str.find('\n\nBOTTOM 10')
+top_250 = file_str[start_point:end_point:]
+lst_top_250 = top_250.split('\n')
+for i in range(len(lst_top_250)):
+    lst_top_250[i] = lst_top_250[i].lstrip()
+dct_top_250 = {}
+top250_str = ''
+dct_ratings = {}
+dct_years = {}
+for el in lst_top_250:
+    el = el.replace('  ', ' ')
+    el = el.replace('  ', ' ')
+    inner_start_point = el.find('.') + 3
+    inner_end_point = el.find('(')
+    top250_str += el[inner_start_point:inner_end_point - 1:] + '\n'
+    str_info = el.replace(el[inner_start_point:inner_end_point - 1:], '')
+    lst_info = str_info.split()
+    lst_info[3] = lst_info[3].replace('(', '').replace(')', '')
+    if dct_ratings.get(lst_info[2]) is None:

```

Comment:
what are these 2, 3 magic numbers?

Reply:
It's position of year and rank date in list